fastseq_cli/transformers_generate.py

```diff
@@ -160,6 +162,9 @@ def generate_summaries_or_translations(
         import fastseq  #pylint: disable=import-outside-toplevel
     fout = Path(out_file).open("w", encoding="utf-8")
     model_name = str(model_name)
+    if model_name == 'prophetnet':
+        model = AutoModelForSeq2SeqLM.from_pretrained(model_name, from_tf=True).to(device)
```

Comment:
Line 173 also uses `AutoModelForSeq2SeqLM`. Could we combine these two?

Reply:
combined